trestle/core/commands/author/ssp.py

```diff
@@ -143,9 +145,19 @@ def _generate_ssp_markdown(
         context.overwrite_header_values = overwrite_header_values
         context.allowed_sections = None
         context.comp_def_name_list = compdef_name_list
-        # this will raise exception if problem loading profile
-        fetcher = FetcherFactory.get_fetcher(trestle_root, str(profile_path))
-        context.profile, _ = fetcher.get_oscal()
+
+        # if file not recognized as URI form, assume it represents name of file in trestle directory
+        profile_in_trestle_dir = '://' not in profile_name_or_href
+        profile_href = profile_name_or_href
+        if profile_in_trestle_dir:
+            local_path = f'profiles/{profile_name_or_href}/profile.json'
+            profile_href = const.TRESTLE_HREF_HEADING + local_path
+            profile_path = trestle_root / local_path
+            _, _, context.profile = ModelUtils.load_distributed(profile_path, trestle_root)
+        else:
+            fetcher = FetcherFactory.get_fetcher(trestle_root, profile_href)
+            context.profile: prof.Profile = fetcher.get_oscal(profile_path)
```

Comment:
Small error profile_path does not exist here just yet, I think you meant profile_href

Reply:
good catch - it should actually just be get_oscal() because the fetcher knows the href.  The optional argument passed in just says whether to force an update of the cached object.